src/org/labkey/test/tests/GpatPlateTemplateTest.java

```diff
@@ -0,0 +1,167 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.Assays;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.util.APIAssayHelper;
+import org.labkey.test.util.DataRegionTable;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Assays.class, DailyB.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 7)
+public class GpatPlateTemplateTest extends BaseWebDriverTest
+{
+    public static final File TEST_PLATE_DATA = TestFileUtils.getSampleData("GPAT/plateData.xlsx");
+    public static final File TEST_PLATE_METADATA = TestFileUtils.getSampleData("GPAT/plate-metadata-1.json");
```

Comment:
these variables can don't need to be referenced outside of the class so can be private